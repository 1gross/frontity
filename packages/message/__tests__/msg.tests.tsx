import msg from "..";

describe("msg", () => {
  test("should output a string in the correct format", () => {
    expect(msg("Something bad happened!")).toMatchSnapshot();
  });
});
