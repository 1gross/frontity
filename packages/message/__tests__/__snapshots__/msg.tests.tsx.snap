// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`msg should output a string in the correct format 1`] = `
"Something bad happened!

If you have any questions, join our community at https://community.frontity.org/.
"
`;
